src/arrayvec.rs

```diff
@@ -640,12 +640,25 @@ impl<T, const CAP: usize> ArrayVec<T, CAP> {
         }
     }
 
+    /// Return the inner fixed size array.
+    ///
+    /// Safety:
+    /// This operation is safe if and only if length equals capacity.
     pub unsafe fn into_inner_unchecked(self) -> [T; CAP] {
         let self_ = ManuallyDrop::new(self);
```

Comment:
Let's put in a debug assertion in this one by the way, to make it debug-checked. I or you can add it before release.

Reply:
Good point, will get that added now